_data/conferences.yml

```
@@ -1,3 +1,16 @@
+- title: ICML
+  year: 2025
+  id: icml25
+  link: https://icml.cc/Conferences/2025
+  deadline: '2024-01-31 23:59:59'
```

```suggestion
  deadline: '2024-01-30 23:59:59'
```
Deadline is 3:59am on Jan 31st, i.e. Jan 30th just before midnight AoE